.gen/go/shared/types.go

```diff
@@ -23098,6 +23098,11 @@ type WorkflowExecutionConfiguration struct {
 	ExecutionStartToCloseTimeoutSeconds *int32       `json:"executionStartToCloseTimeoutSeconds,omitempty"`
 	TaskStartToCloseTimeoutSeconds      *int32       `json:"taskStartToCloseTimeoutSeconds,omitempty"`
 	ChildPolicy                         *ChildPolicy `json:"childPolicy,omitempty"`
+	StickyTaskList                      *TaskList    `json:"stickyTaskList,omitempty"`
+	StickyScheduleToStartTimeoutSeconds *int32       `json:"stickyScheduleToStartTimeoutSeconds,omitempty"`
+	ClientLibraryVersion                *string      `json:"clientLibraryVersion,omitempty"`
+	ClientFeatureVersion                *string      `json:"clientFeatureVersion,omitempty"`
+	ClientLang                          *string      `json:"clientLang,omitempty"`
```

Comment:
i'm not sure we want to public expose these to users.

Reply:
removed

renamed getworkflow next event id to get mutable state, which returns these things.